pkg/daemons/agent/agent_windows.go

```diff
@@ -0,0 +1,6 @@
+// +build windows
+
+package agent
+
+func createRootlessConfig(argsMap map[string]string, hasCfs, hasPIDs bool) {
+}
```

Comment:
Should this function instead be wrapped above with an os check? Made a comment above

Reply:
I chose to do it with an alternative implementation as we will probably need to implement something windows specific. It also seems that using an OS check isn't as reliable as it appears. Happy to change if we want to follow a different convention.

Reply:
This sounds good to me. @luthermonson informed me that this function will probably include more later.

Reply:
**Resolved**